### DISCUSSION TITLE
IDEA - to have a user-defined script for regular file transfers

### DISCUSSION THREAD

**Discussion by daverep** [iterate-ch/cyberduck#16011] [Ideas]
This would be something like Microsoft scripts or AppleScript or a shell to 1. define the source and destination platforms - then 2. to define a source URL / folder path - a mask for a file(s) and a destination URL path - then there could be a completion step upon successful completion of the file transfer. A scheduler timer would be fantastic for automatic transfers - a LOG would certainly be much appreciated. thx for the forum.

**Comment by dkocher** [comment]
Previously in #1852.
